src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -721,5 +721,36 @@ void GcsActorManager::SchedulePendingActors() {
   }
 }
 
+void GcsActorManager::LoadInitialData(const StatusCallback &done) {
+  RAY_LOG(INFO) << "Loading initial data.";
+  auto callback = [this,
+                   done](const std::unordered_map<ActorID, ActorTableData> &result) {
+    for (auto &item : result) {
+      if (item.second.state() != ray::rpc::ActorTableData::DEAD) {
+        auto actor = std::make_shared<GcsActor>(item.second);
+        registered_actors_.emplace(item.first, actor);
+
+        if (actor->IsDetached()) {
+          named_actors_.emplace(actor->GetName(), actor->GetActorID());
+        }
+
+        created_actors_[actor->GetNodeID()].emplace(actor->GetWorkerID(),
+                                                    actor->GetActorID());
+
+        auto &workers = owners_[actor->GetNodeID()];
+        auto it = workers.find(actor->GetWorkerID());
+        if (it == workers.end()) {
+          std::shared_ptr<rpc::CoreWorkerClientInterface> client =
+              worker_client_factory_(actor->GetOwnerAddress());
+          workers.emplace(actor->GetOwnerID(), Owner(std::move(client)));
+        }
+      }
+    }
+    RAY_LOG(INFO) << "Finished loading initial data.";
+    done(Status::OK());
```

Comment:
use an empty callback if the status is always ok.

Reply:
fixed